tests/sasl_test.py

```diff
@@ -211,6 +221,11 @@ def handle_report(report, version, suite):
                'rdkconf': {'sasl_oauthbearer_config': 'scope=wrongScope'},
                'tests': ['0001'],
                'expect_fail': ['all']},
+              {'name': 'OAuth/OIDC',
+               'run': args.oidc,
+               'tests': ['0126'],
```

Comment:
Can we throw in one of the standard tests as well just to verify that the OIDC config works without anything extra needing to be done by the app. I recommend test 0001